src/main/java/stream/flarebot/flarebot/commands/moderation/RolesCommand.java

```diff
@@ -58,7 +59,15 @@ public void onCommand(User sender, GuildWrapper guild, TextChannel channel, Mess
             }
 
             sb.append("```\n").append("**Page ").append(GeneralUtils.getPageOutOfTotal(page, roles, pageSize)).append("**");
-            MessageUtils.sendInfoMessage(sb.toString(), channel, sender);
+            MessageUtils.sendInfoMessage(sb.toString(), channel, sender);*/
+
+            StringBuilder sb = new StringBuilder();
+            for(Role r : guild.getGuild().getRoles())
+                sb.append(r.getName()).append(" (").append(r.getId()).append(")\n");
+            int page = 1;
+            if(args.length == 1)
+                page = Integer.parseInt(args[0]);
+            //PaginationUtil.sendPagedMessage(channel, sb.toString(), PaginationUtil.SplitMethod.CHAR_COUNT, 1000, page, true);
```

Comment:
Why is the commented?

Reply:
From testing. give me a min to clean that up